lms/djangoapps/course_api/blocks/tests/test_api.py

```diff
@@ -77,3 +77,32 @@ def test_sub_structure(self):
                 self.assertIn(unicode(block.location), blocks['blocks'])
             else:
                 self.assertNotIn(unicode(block.location), blocks['blocks'])
+
+    def test_visible_to_staff_only(self):
```

Comment:
Note: It may not have been clear from the code or the (missing) documentation on the testing convention, but in order to minimize duplication of test code, we have opted to segregate the testing between `test_serializers.py`, `test_views.py`, and `test_api.py`.  In particular, test_api.py is focused on testing access-specific tests and parameters that are only in api.py.  Whereas, `requested_fields` is currently tested in `test_serializers.py`.

Please take a look at extending [add_additional_requested_fields](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/course_api/blocks/tests/test_serializers.py#L61) and [assert_extended_block](https://github.com/edx/edx-platform/blob/master/lms/djangoapps/course_api/blocks/tests/test_serializers.py#L75) in `test_serializers.py` instead of adding a test method in this module.